server/src/test/java/io/druid/server/coordination/CostBalancerStrategyTest.java

```diff
@@ -156,4 +160,32 @@ public void testBenchmark() throws InterruptedException
     System.err.println("Latency - Single Threaded (ms): " + latencySingleThread);
     System.err.println("Latency - Multi Threaded (ms): " + latencyMultiThread);
   }
+
+    @Test
+    public void testComputeJointSegmentCost()
+    {
+      DateTime referenceTime = DateTime.now(DateTimeZone.UTC);
```

Comment:
I would prefer if we used specific timestamps in tests to ensure they are deterministic.